core/dbt/task/run.py

```diff
@@ -398,11 +398,15 @@ def print_results_line(self, results, execution_time):
         execution = ""
 
         if execution_time is not None:
-            execution = " in {execution_time:0.2f}s".format(execution_time=execution_time)
+            execution = utils.humanise_execution_time(execution_time=execution_time)
```

Comment:
I think for now it's fine to leave the prettification it where it is. I suspect that HookFinished may be one of the events that we'll be touching with the upcoming event work, so rather than guess what info people will want in it, let's just defer it :)

I think a test in test/unit/test_utils.py would be fine. You could create a basic test that does 'run_dbt_and_capture' (grep tests/functional for examples). I'm not sure exactly where to put it, but someplace neutral like tests/functional/basic/test_jaffle_shop.py should be fine.

Reply:
Thanks for the guidance @gshank. I've gone ahead and done those things. Let me know what you think.